modules/user-provided-certificates-for-default-ingress.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="user-provided-certificates-for-default-ingress_{context}"]
+= User-provided certificates for default ingress
```

Comment:
@sanchezl Can you please review what I have in this section so far?

Reply:
I would expect this one to be @ironcladlou